operator/cmd/mesh/manifest-generate_test.go

```diff
@@ -195,6 +197,74 @@ func TestManifestGenerateGateways(t *testing.T) {
 	}
 }
 
+func TestManifestGenerateWithDuplicateMutatingWebhookConfig(t *testing.T) {
+	testResourceFile := "duplicate_mwc"
+
+	testCases := []struct {
+		name       string
+		force      bool
+		assertFunc func(g *WithT, objs *ObjectSet, err error)
+	}{
+		{
+			name:  "Duplicate MutatingWebhookConfiguration should not be allowed when --force is enabled",
+			force: true,
+			assertFunc: func(g *WithT, objs *ObjectSet, err error) {
+				g.Expect(err).Should(BeNil())
+				g.Expect(objs.kind(name.MutatingWebhookConfigurationStr).size()).Should(Equal(2))
+			},
+		},
+		{
+			name:  "Duplicate MutatingWebhookConfiguration should not be allowed when --force is disabled",
+			force: false,
+			assertFunc: func(g *WithT, objs *ObjectSet, err error) {
+				g.Expect(err.Error()).Should(BeEquivalentTo("creating default tag would conflict:\nError [IST0139] (MutatingWebhookConfiguration" +
```

Comment:
can we just do some strings.Containers match for 'Webhook overlaps with others'? We don't need to test the exact output here, may be hard to maintain

Reply:
makes sense, let me fix that.